src/components/ErrorBoundary/index.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import { mount } from 'enzyme';
+import * as React from 'react';
+
+import { createFakeLogger } from '../../test-helpers';
+import styles from './styles.module.scss';
+
+import ErrorBoundary, { Props } from '.';
+
+describe(__filename, () => {
+  const CustomComponent = ({
+    children = <div />,
+  }: {
+    children?: JSX.Element;
+  }) => children;
+
+  type RenderParams = Partial<Props> & { children?: JSX.Element };
```

Comment:
`children` should be a required public prop of the actual component because it would be a mistake to render `<ErrorBoundary />` without any children.